mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/kafka/consumers/amt/AmtVirksomheterV1TopicConsumer.kt

```diff
@@ -22,13 +23,17 @@ class AmtVirksomheterV1TopicConsumer(
     private val logger = LoggerFactory.getLogger(javaClass)
 
     override suspend fun consume(key: String, message: JsonElement) {
+        virksomhetRepository.get(key).getOrThrow() ?: return
```

Comment:
Kunne dette vært en funksjon med et beskrivende navn? Kan virke litt tilfeldig å forstå hvorfor denne linjen er her.

Reply:
Jau, tenkte på det samme. Endte opp med dette fordi jeg ikke kjenner til noe kotlin-funksjonalitet for implisitt return via et nytt funksjonskall. Kan gjøre intensjonen det litt mer tydelig :+1:

Reply:
Ah, skjønner. Kanskje en kommentar i hvert fall gjør det litt tydeligere